src/basics.md

```diff
@@ -1092,6 +1093,35 @@ fn main() {
 }
 ```
 
+[ex-format-datetime]: #ex-format-datetime
+<a name="ex-format-datetime"></a>
+## Display formatted date and time
+[![chrono-badge]][chrono] [![cat-date-and-time-badge]][cat-date-and-time]
+
+Displays UTC in the well-known formats RFC 2822 and RFC 3339, and in a custom
+format.
+
+```rust
+extern crate chrono;
+use chrono::{DateTime, Utc};
+
+fn main() {
+    // Get coordinated universal time (UTC)
+    let utc: DateTime<Utc> = Utc::now();
+    println!("UTC is: {}", utc);
+
+    // Display formatted date and time using RFC 2822
+    println!("UTC in RFC 2822 is: {}", utc.to_rfc2822());
+
+    // Display formatted date and time using RFC 3339
+    println!("UTC in RFC 3339 is: {}", utc.to_rfc3339());
+
+    // Display formatted date and time using a custom format
+    const CUSTOM_FORMAT: &str = "%a %b %e %T %Y";
```

Comment:
Probably we do not need a separate variable here. Let's keep the example as simple as possible!